tests/posix/fs/src/test_fat_mount.c

```diff
@@ -37,7 +49,6 @@ static int test_mount(void)
  * @details Test initializes the fs_mount_t data structure with FatFs
  * related info and calls the fs_mount API for mount the file system.
  */
-void test_fs_mount(void)
+ZTEST(posix_fs_test, test_fs_mount)
 {
-	zassert_true(test_mount() == TC_PASS, NULL);
```

Comment:
Is there nothing we can assert here?

Reply:
I found something - it's reaching into implementation details a bit, but should work for this test.